src/components/navbar/SealVerse.tsx

```diff
@@ -0,0 +1,17 @@
+import Dropdown from 'components/Dropdown'
+import sealVerseData from 'sealVerseData'
+
+export default function () {
+  return (
+    <Dropdown
+      currentValue={window.location.origin}
+      options={sealVerseData}
+      staticPlaceholder="SealVerse"
+      onChange={(value) => {
+        if (value && value !== window.location.origin)
```

Comment:
What if the user will use the `dropdown` on `sealcaster.xyz/app`? Will this work? Maybe we should do `window.location.origin.includes(value)`?